python/tests/test_conftest.py

```diff
@@ -383,3 +383,18 @@ def m(a, b, c, d):
             assert len(items) == 1
             res = pytester.runpytest()
             assert res.ret == 0
+
+
+    def test_time(self, pytester: Pytester):
+        checkfile = pytester.makepyfile(
+    """
+        from inline import Here
+        def m(a):
+            c = [i + 3 for i in a]
+            Here(timeout=1).given(a, [i + 1 for i in range(1000000)]).check_eq(len(c), len(a))
+    """)
+        for x in (pytester.path, checkfile):
+            items, reprec = pytester.inline_genitems(x)
+            assert len(items) == 1
+            res = pytester.runpytest()
+            assert res.ret == 0
```

Comment:
Thanks for the test. It is better to add one failing test to check if TimeoutException is thrown out